src/index.tsx

```diff
@@ -25,6 +26,14 @@ const App = () => {
                     Pods Logs
                   </Link>
                 </li>
+                <li className="pf-c-nav__item">
+                  <Link
+                    className="pf-c-nav__link"
+                    to="/dev-monitoring/ns/my-namespace/logs"
```

Comment:
Why do we hardcode this to be `my-namespace`? (I see that `test-pod-name` is also hardcoded above, so I guess this is intentional.)

Reply:
Yes, is intentional as this plugin runs in a test standalone mode to run the e2e tests without the need of a cluster/console as is only testing the components behavior and not the integration.